Corzbank.Data/Entities/TransferCard.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Corzbank.Data.Entities
+{
+    public class TransferCard: KeyEntity
+    {
+        public int TransferId { get; set; }
+        public Transfer Transfer { get; set; }
+
+        public int SenderCardId { get; set; }
+        public int ReceiverCardId { get; set; }
```

Comment:
This key will be null in case we want to transfer to phone number. Add constraints to all the places as we discussed [here](https://docs.google.com/document/d/1fZO7oABSkYbb0I0OkiHrQk-Z7eEUMHNLiR0AeaFbQhg/edit), maybe even more and update migration

Reply:
ReceiverCardId must be nullable

Reply:
resolved